test/saithrift/Makefile

```diff
@@ -3,11 +3,17 @@ CXX = $(CROSS_COMPILE)g++
 SAI_PREFIX = /usr
 SAI_HEADER_DIR ?= $(SAI_PREFIX)/include/sai
 SAI_HEADERS = $(SAI_HEADER_DIR)/sai*.h
-CFLAGS = -I$(SAI_HEADER_DIR) -I. -std=c++11 -DFORCE_BOOST_SMART_PTR
+CFLAGS = -I$(SAI_HEADER_DIR) -I. -std=c++11
 ifeq ($(DEBUG),1)
 CFLAGS += -O0 -ggdb
 endif
 
+# Detect THRIFT_VERSION
+THRIFT_VERSION = $(shell thrift  -version | cut -d ' ' -f3)
+ifeq ($(shell dpkg --compare-versions $(THRIFT_VERSION) "le" 0.11.0 && echo True), True)
```

Comment:
In order not to break the compatibility with the test cases using the saiserver in syncd-rpc which depends on the thrift client in PTF with version in 0.11, I make another version of saiserver in this PR https://github.com/opencomputeproject/SAI/pull/1388, do we really need to upgrade the saiserver and the dependent thrift for previous test cases?

Reply:
In case we need to run syncd-rpc with original [saiserver](https://github.com/opencomputeproject/SAI/tree/master/test/saithrift). For example, it is needed for sonic-mgmt [QOS test](https://github.com/Azure/sonic-mgmt/tree/master/tests/qos). For the Barefoot platform(maybe others) there is the dependency on the newer thrift version. In this PR I do not change the API or saiserver behavior. Just make it compatible with thrift 0.14.1 and keep backward compatibility with thrift 0.11.0 to make the transition period as simple as possible. So I do not break compatibility with older test cases. Once it will be merged and refpoint updated in SONiC then vendors will be able to choose which thrift version to use.

Reply:
thanks for the information, actually, I want to confirm if this change will compatible with thrift 0.11.0. 
I think I got my answer.

thanks.